packages/api/src/external/commonwell/admin/find-patient-duplicates.ts

```diff
@@ -182,18 +180,21 @@ export async function findDuplicatedPersonsByPatient(
 
     // for each person, get their enrolment details
     for (const person of filteredPersons) {
-      const personId = getId(person);
+      const personId = getPersonId(person);
       if (!personId) {
         console.log(
           `Got a person without ID for patient ${patient.id}, skipping - ${stringify(person)}`
         );
         continue;
       }
-      const patientLinks = await commonWell.getPatientLinks(queryMeta, personId);
+      const patientLinks = await commonWell
+        .getPatientLinks(queryMeta, personId)
+        .catch(simpleErrorLog(`getPatientLinks, person ${personId}`));
       res[personId] = {
-        amountOfLinks: patientLinks._embedded?.patientLink?.length || 0,
-        enroller: person.enrollmentSummary?.enroller || "unknown",
-        enrollmentDate: person.enrollmentSummary?.dateEnrolled || "unknown",
+        amountOfLinks: patientLinks?._embedded?.patientLink?.length ?? indetermined,
+        enrolled: person.enrolled ?? indetermined,
+        enroller: person.enrollmentSummary?.enroller ?? indetermined,
+        enrollmentDate: person.enrollmentSummary?.dateEnrolled ?? indetermined,
```

Comment:
is `??` what we want here? if the string is empty, we'll set it to `""` rather than `undetermined`

Reply:
good catch